### PR TITLE
Update golang and fluent-bit versions in Dockerfile

### PR DESCRIPTION
**Pull request by mtparet**
* Fix fluent-bit version to know which version if embedded

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.4-buster AS build-env
+FROM golang:1.13.8-buster AS build-env
```
Comment (cosmo0920) [inline Dockerfile]: Why don't you use golang 1.14 image?

File: Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ ADD . /go/src/github.com/cosmo0920/fluent-bit-go-s3
 WORKDIR /go/src/github.com/cosmo0920/fluent-bit-go-s3
 RUN make build
 
-FROM fluent/fluent-bit:1.3
```
Comment (cosmo0920) [inline Dockerfile:10]: fluent/fluent-bit:1.3 always points to the v1.3 series latest image.
If you want to rebuild per the latest fluent-bit 1.3 series, how about the following?

```
LABEL Description="Fluent Bit Go S3" FluentBitVersion="1.3.9"
```

**Review by cosmo0920 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mtparet) [reply Dockerfile]: indeed we could bump directly

File: Dockerfile (hunk 2 above)
Comment (mtparet) [reply Dockerfile:10]: We could add the label too indeed, specifying the version in the `FROM` ensure we are effectively using the right version.

Comment (cosmo0920) [reply Dockerfile:10]: Make sense.

**Review by cosmo0920 — APPROVED** [review]

**Pull request merged**